src/infrastructure/data-access/typeorm/room.entity.ts

```diff
@@ -61,7 +63,17 @@ export class RoomModel {
   })
   isActive: boolean;
 
+  @Column({
+    name: 'status_id',
+    nullable: true,
+  })
+  statusId: number;
+
   @ManyToOne(() => UserModel, (user) => user.rooms)
   @JoinColumn({ name: 'owner' })
   user?: UserModel;
+
+  @OneToMany(() => StatusModel, (status) => status.room)
+  @JoinColumn({ name: 'status_id' })
+  confirmationBookingStatuses?: StatusModel[];
```

Comment:
Change this field name to `roomStatus`
Why this field has type is `StatusModel[]`? Should be `StatusModel` because the relationship between `room` and `status` is `many to one`